scipy/constants/_constants.py

```diff
@@ -7,8 +7,6 @@
 The list is not meant to be comprehensive, but just convenient for everyday use.
 """
 
-from __future__ import annotations
```

Comment:
that `TYPE_CHECKING` import below looks like trouble...

Reply:
added quotes here